.travis.yml

```diff
@@ -2,4 +2,4 @@ language: node_js
 node_js:
   - "4.2"
   - "5.1"
-script: npm run lint
+script: gulp lint && gulp build
```

Comment:
I don't think this will work since gulp isn't installed globally. Probably need to stick with `npm run lint && npm run build`.